pkg/pkg.ml

```diff
@@ -33,7 +33,10 @@ let () =
     Ok [ Pkg.lib "pkg/META.mirage-types" ~dst:"META";
          Pkg.lib "types/V1.mli";
          Pkg.lib "types/V1.cmi";
+         Pkg.lib "types/V1.cmti";
          Pkg.lib ~cond:lwt "types/V1_LWT.mli";
-         Pkg.lib ~cond:lwt "types/V1_LWT.cmi"; ]
```

Comment:
Are these mli only compilation units ? I should introduce an `Exts.types_api` shortcut for these.